README.md

```diff
@@ -101,7 +101,7 @@ Features
 Prerequisites
 -------------
 
-- [MongoDB](https://www.mongodb.org/downloads)
+- [MongoDB](https://www.mongodb.org/downloads) needs to be installed and running.
```

Comment:
The user doesn't have to download, install, and run MongoDB, since they can use an mLab's hosted database. 

The Readme sections: "Tip" under FAQ -> "I am getting MongoDB Connection Error, how do I fix it?"  and the section about using MongoLab.

Reply:
That's fair!